### DISCUSSION TITLE
javascript-for-net-developers-march-2015-edition/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#50] [Blog comments]
# javascript-for-net-developers-march-2015-edition/

https://blogs.taiga.nl/martijn/posts/javascript-for-net-developers-march-2015-edition/

**Comment by disqus-comments-migrator** [comment]
<em>[Mladen Mihajlovic](https://disqus.com/by/mika76/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at maart 02 2016, 07:53</em>

---

<p>This was a great compilation of important facts, how about a 2016 version?</p>

**Reply by martijnboland** [reply]
<em>[Martijn Boland](https://disqus.com/by/martijnboland/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at maart 02 2016, 09:17, in reply to [Mladen Mihajlovic](https://disqus.com/by/mika76/)'s [comment](https://github.com/martijnboland/blog/discussions/50#discussioncomment-8952950)</em>

---

<p>Good idea, although things haven't changed that drastically last year framework-wise</p>
